examples/trl_example/stack_llama_2/README.md

```diff
@@ -0,0 +1,36 @@
+# DPO pipeline for the creation of StackLlaMa 2: a Stack exchange llama-v2-7b model
+
+## Prerequisites
+
+Install all the dependencies in the `requirements.txt`:
+
+```
+$ pip install -U -r requirements.txt
+```
+
+## Training
+
+There were two main steps to the DPO training process:
+1. Supervised fine-tuning of the base llama-v2-7b model to create llama-v2-7b-se:
+    - `python ../../gaudi_spawn.py --world_size 8 --use_mpi sft_llama2.py --training_args.output_dir="sft_output" --training-args.report_to none`
```

Comment:
i saw there are many passed-in arguments from [example/trl/stack_llama_2](https://github.com/huggingface/trl/tree/main/examples/research_projects/stack_llama_2/scripts . these are not needed?

Reply:
I fixed it in https://github.com/huggingface/optimum-habana/pull/600, make sure your codebase contains this fix